nxrm-ha/README.md

```diff
@@ -320,6 +321,244 @@ Azure Key Vault is disabled by default. If you would like to store your database
                                     name: nexus-repository-dev-ha-sa # use same service account name as specified in serviceAccount.name
                   ```
                   
+### GCP
+
+ * Update values.yaml
+   1. Enable service account  (you’ll need the name of the existing Google service account - there is a chapter below on how to create it)
+  
+       ```
+         serviceAccount:
+           enabled: true
+           name: nexus-repository-deployment-sa
+           labels: {}
+           annotations:
+             meta.helm.sh/release-name: <your-release-name>
+             meta.helm.sh/release-namespace: nexusrepo
+             iam.gke.io/gcp-service-account: <service-account-name>@<project-id>.iam.gserviceaccount.com <- your GCP project service account e-mail
+        
+       ```
+
+   2. Enable ingress 
+        ```
+        ingress:
+          name: "nexus-ingress"
+          enabled: true
+        ```  
+   3. Update ingress properties to correspond to GKE controller class https://cloud.google.com/kubernetes-engine/docs/how-to/load-balance-ingress#create-ingress 
+        ```
+        ingress:
+          name: "nexus-ingress"
+          enabled: true
+          ...
+          defaultRule: true
+          additionalRules: null
+          ingressClassName: gce
+          ...
+          annotations:
+            kubernetes.io/ingress.class: "gce"
+        ```
+   4. Enable Nexus NodePort service 
```

Comment:
Is it possible to use ClusterIP as well on GKE? If so consider changing to 'Enable Nexus NodePort/ClusterIP service'

Reply:
yes, updated